data/INRA/2016_harvest_data.ttl

```diff
@@ -0,0 +1,53 @@
+ <data/winemaking/PechRouge/Harvest/2016> a qb:DataSet; 
+  rdfs:label "Data about Harvest done on Pech Rouge in 2016"; 
+  qb:structure <DSD/harvestWeight/2016> . 
+ 
+<DSD/harvestWeight/2016> a qb:DataStructureDefinition; 
+  qb:component [qb:dimension bdg:plot                                                              ]; 
+  qb:component [qb:dimension bdg:variety                                                           ]; 
+  qb:component [qb:dimension bdg:date                                                              ]; 
+  qb:component [qb:dimension bdg:humanSensor                                                       ]; 
+  qb:component [qb:measure   bdg:grapeClusterWeight    ; qb:componentAttachment qb:DataSet         ]; 
+  qb:component [qb:attribute sosa:hasFeatureOfInterest ; qb:componentAttachment qb:MeasureProperty ]; 
+  qb:component [qb:attribute sdmx-attribute:unitMeasure; qb:componentAttachment qb:MeasureProperty ] . 
+ 
+<variety/Chardonnay> a bdg:Variety; 
```

Comment:
Look in the [model](https://github.com/BigDataGrapes-EU/ontology/tree/master/model#2131-measurement-contexts) how to build the codelists.